dashboard/app/helpers/course_block_helper.rb

```diff
@@ -169,13 +169,55 @@ def self.get_tall_course_block(id)
       Script::TEXT_COMPRESSION_NAME => {
         url: script_reset_url(id),
         teacher_guide: "https://curriculum.code.org/hoc/2016/1/"
-      },
-      Script::TWENTY_HOUR_NAME => {
-        title: I18n.t('upsell.20hour.title'),
-        body: I18n.t('upsell.20hour.body')
       }
     }
 
     default.merge!(block_override_data[id] || {})
   end
+
+  #
+  # Given a Script ID or a string, returns an object that represents
+  # a wide course block (_course_wide_block.haml), if one exists.
+  #
+  # Example return value:
+  # {
+  #   url: "https://my.url",
+  #   title: "Wide Course Block",
+  #   body: "Click to find out more!",
+  #   image_url: "https://my.image",
+  #   audience: "Pre-reader"
+  # }
+  #
+  def self.get_wide_course_block(id)
+    block_override_data = {
+      Script::EXPRESS_NAME => {
+        url: script_url(id),
+        title: data_t_suffix('script.name', id, 'title'),
+        body: data_t_suffix('script.name', id, 'description_short'),
+        image_url: CDO.code_org_url("/shared/images/courses/fill-150x110/logo_tall_#{id}.jpg")
```

Comment:
Unnecessary, but it would be possible to factor out the `fill-150x110` bit if we wanted, since it's a common adjustment we make to each of these URLs.